xlml/apis/test_config.py

```diff
@@ -480,10 +476,45 @@ class JSonnetGpuTest(TestConfig[Gpu]):
   """
 
   test_name: str
-  test_command: List[str]
   entrypoint_script: List[str]
+  test_command: List[str]
   docker_image: str
+  accelerator: Gpu
+  time_out_in_min: str
```

Comment:
timeout should be a `timedelta` so the unit is self-describing

Reply:
I mean the type of this should be `datetime.timedelta`. Calling the variable "timeout` is fine